lib/bootloader_pvm.pm

```diff
@@ -131,6 +131,12 @@ sub enter_netboot_parameters {
     type_string_slow remote_install_bootmenu_params;
     type_string_slow " fips=1" if (get_var('FIPS_INSTALLATION'));
     type_string_slow " UPGRADE=1" if (get_var('UPGRADE'));
+
+    if (get_var('AGAMA_TEST')) {
+        my $host = get_var('OPENQA_HOSTNAME', 'openqa.opensuse.org');
```

Comment:
isn't always a value for this variable? I don't know why to set a default.
I think if you don't have that, you don't need to declare explicitly two variables, you could do it in one line, the less assignation sometimes the better, it might help to debug in some cases, but not this one.

Reply:
817b033f2